src/validations.ts

```diff
@@ -1,13 +1,39 @@
-import { LogLevel } from './logs';
-import { IPathPosition } from './parsers';
+import { IPathPosition, ObjPath } from './parsers';
+
+export enum ValidationSeverity {
+  Info = 'info',
+  Warn = 'warn',
+  Error = 'error',
+}
+
+export enum ValidationSeverityNum {
+  Info = 30,
+  Warn = 40,
+  Error = 50,
+}
 
 export interface IValidation {
```

Comment:
This seems more an instance of a Validation, not a Validation specification. Perhaps `IValidationResult` might be a better name for this.

Reply:
I agree, will update.